packages/search-ui/src/types/index.ts

```diff
@@ -50,9 +50,10 @@ export type RequestState = {
   filters?: Filter[];
   resultsPerPage?: number;
   searchTerm?: string;
-  sortDirection?: SortDirection;
-  sortField?: string;
-  sortList?: SortOption[];
+  sortDirection?: SortDirection; // Deprecated
+  sortField?: string; // Deprecated
+  sortList?: SortOption[]; // Maybe deprecated? Not in docs
+  sort?: SortOption[];
```

Comment:
yeh what was obvious whilst i was doing sorting types, it was a mixture of old and new. When we mature our TS conversion over next coming weeks, definitely need to zoom into sorting options and document them well.